### PR TITLE
Frontend fully conected to backend

### PR DESCRIPTION
**Pull request by Bailonis**
In this pr we did all the connection from the backend to the frontend, so it may have some big changes, but from now one, the changes would be small

### CODE REVIEW

**Review by SebastianBezold — CHANGES_REQUESTED** [review]

**Comment by SebastianBezold** [comment]
Hi @Bailonis , i have some comments about the changes, please have a look at them.
Also, it would be great if someone from your team could do a content review, since I cannot answer things like "are all wanted features included", "are all the unit-tests added"
See also: [Developer Hub: Code Reviews](https://eclipse-tractusx.github.io/docs/oss/code-reviews)

**Comment by SebastianBezold** [comment]
I created a `vet-project` IP issue to check the PR, since is quite big: https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/10164.

File: .github/workflows/veracode-pipeline.yml [hunk 1]
```diff
@@ -18,14 +18,13 @@
 #   SPDX-License-Identifier: Apache-2.0
 #   ********************************************************************************
 
-
 name: Veracode Static Analysis Pipeline Scan
 
 on:
   push:
-    branches: [ "main" ]
```
[the remark this answers is not in the document]
Comment (Bailonis) [reply .github/workflows/veracode-pipeline.yml]: Done

File: demand-capacity-mgmt-backend/src/main/java/org/eclipse/tractusx/demandcapacitymgmt/demandcapacitymgmtbackend/entities/enums/CapacityGroupStatus.java
[the remark this answers is not in the document]
Comment (Bailonis) [reply demand-capacity-mgmt-backend/src/main/java/org/eclipse/tractusx/demandcapacitymgmt/demandcapacitymgmtbackend/entities/enums/CapacityGroupStatus.java:1]: Done

File: demand-capacity-mgmt-backend/src/main/java/org/eclipse/tractusx/demandcapacitymgmt/demandcapacitymgmtbackend/entities/enums/MaterialDemandStatus.java
[the remark this answers is not in the document]
Comment (Bailonis) [reply demand-capacity-mgmt-backend/src/main/java/org/eclipse/tractusx/demandcapacitymgmt/demandcapacitymgmtbackend/entities/enums/MaterialDemandStatus.java:1]: Done
image

File: .github/workflows/veracode-pipeline.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Bailonis) [reply .github/workflows/veracode-pipeline.yml]: image

File: demand-capacity-mgmt-frontend/src/components/demands/DemandPage.tsx
[the remark this answers is not in the document]
Comment (Bailonis) [reply demand-capacity-mgmt-frontend/src/components/demands/DemandPage.tsx:1]: done
image

**Comment by Ruskyy** [comment]
Nothing to point from my side.
Other than the nice finds from @SebastianBezold.
Unit Tests and Features were already pre discussed in code review, and are approved !

**Comment by SebastianBezold** [comment]
IP issue resolved without findings! Will merge

**Pull request merged**